{"name": "test_log_in_failed", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'ERROR: Incorrect username or password.' in 'ERROR: Too many failed login attempts. Please try again in 11 minutes.'\n +  where 'ERROR: Too many failed login attempts. Please try again in 11 minutes.' = <bound method MyAccountPage.get_error_msg of <page_object_patern.pages.my_account_page.MyAccountPage object at 0x000001DD8B2EB910>>()\n +    where <bound method MyAccountPage.get_error_msg of <page_object_patern.pages.my_account_page.MyAccountPage object at 0x000001DD8B2EB910>> = <page_object_patern.pages.my_account_page.MyAccountPage object at 0x000001DD8B2EB910>.get_error_msg", "trace": "self = <page_object_patern.tests.login_test.TestLogIn object at 0x000001DD8B24B390>\n\n    def test_log_in_failed(self):\n        my_account_page = MyAccountPage(self.driver)\n        my_account_page.open_page()\n        my_account_page.log_in(\"sialalala1@gmail.coml\", \"Tester12345!@#$%\")\n    \n>       assert \"ERROR: Incorrect username or password.\" in my_account_page.get_error_msg()\nE       AssertionError: assert 'ERROR: Incorrect username or password.' in 'ERROR: Too many failed login attempts. Please try again in 11 minutes.'\nE        +  where 'ERROR: Too many failed login attempts. Please try again in 11 minutes.' = <bound method MyAccountPage.get_error_msg of <page_object_patern.pages.my_account_page.MyAccountPage object at 0x000001DD8B2EB910>>()\nE        +    where <bound method MyAccountPage.get_error_msg of <page_object_patern.pages.my_account_page.MyAccountPage object at 0x000001DD8B2EB910>> = <page_object_patern.pages.my_account_page.MyAccountPage object at 0x000001DD8B2EB910>.get_error_msg\n\npage_object_patern\\tests\\login_test.py:21: AssertionError"}, "attachments": [{"name": "stdout", "source": "fda3b93e-f9b4-4d46-842c-64334ebc0efe-attachment.txt", "type": "text/plain"}], "start": 1683796546572, "stop": 1683796547777, "uuid": "84dbc576-9aab-48ce-a1f0-4e15a247921d", "historyId": "f4b1f1a29402a58bee951e2d821863e0", "testCaseId": "f4b1f1a29402a58bee951e2d821863e0", "fullName": "page_object_patern.tests.login_test.TestLogIn#test_log_in_failed", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "page_object_patern.tests"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestLogIn"}, {"name": "host", "value": "LaptopAsia"}, {"name": "thread", "value": "10368-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "page_object_patern.tests.login_test"}]}